src/dispatch/static/dispatch/src/incident/TimelineTab.vue

```diff
@@ -76,6 +73,26 @@ export default {
       Util.exportCSV(items, this.name + "-timeline-export.csv")
       this.exportLoading = false
     },
+    formatToTimeZones(date) {
```

Comment:
If we think that this will be used elsewhere we could also make this into a filter: https://github.com/Netflix/dispatch/blob/167442773ad7925792c27c2ea4ced59d67154cae/src/dispatch/static/dispatch/src/filters.js#L4